composer/metrics/nlp.py

```diff
@@ -265,11 +265,22 @@ def replace_underscore(text: str) -> str:
 
         return white_space_fix(remove_articles(handle_punc(lower(replace_underscore(answer))))).strip()
 
-    def update(self, outputs: List[str], labels: List[List[str]]):
+    def update(self, batch, outputs: List[str], labels: List[List[str]]):
```

Comment:
`List` and `Dict` are being deprecated in favor of `list` and `dict`. Python 3.9, PEP 585 https://docs.python.org/3/whatsnew/3.9.html#type-hinting-generics-in-standard-collections

Reply:
Also fix elsewhere